com/win32com/HTML/QuickStartServerCom.html

```diff
@@ -12,7 +12,7 @@
 <H1>Quick Start to Server side COM and Python</H1>
 <H2>Introduction</H2>
 <P>This documents how to quickly start implementing COM objects in Python. It is not a thorough discussion of the COM system, or of the concepts introduced by COM.</P>
-<P>For more details information on Python and COM, please see the <A HREF="http://www.python.org/windows/win32com/COMTutorial/index.htm">COM Tutorial given by Greg Stein and Mark Hammond at SPAM 6 (HTML format)</A> or download the same tutorial <A HREF="http://www.python.org/windows/win32com/COMTutorial.ppt">in PowerPoint format.</A></P>
+<P>For more details information on Python and COM, please see the COM Tutorial given by Greg Stein and Mark Hammond at SPAM 6 <A HREF="https://web.archive.org/web/20001117033400/http://www.python.org:80/windows/win32com/COMTutorial/index.htm">(in HTML format, lost to time)</A> or download the same tutorial <A HREF="https://web.archive.org/web/20060511064916/http://www.python.org/windows/win32com/COMTutorial.ppt">in PowerPoint format.</A></P>
```

Comment:
The HTML tutorial seems to be lost to time, since the navigation used JavaScript which wasn't archived.
The PowerPoint however, is still downloadable, and could be added to Pywin32's documentation.

Reply:
heh, I've got that locally :) Not sure there's value to adding it to the repo tbh though, 300 files generated somehow.

Reply:
130 files! :O (I'm guessing some JS navigation library ? Lots of images maybe)

If you think it still has value, it could be hosted as a GitHub page (standalone repo to not pollute pywin32).
The PowerPoint isn't as bad as it's a single 150kb file. Could be hosted right with the existing online docs.

If you think they're so old they're irrelevant and don't bring value anymore, and can be forgotten about, then I'll remove the link(s) instead.